### PR TITLE
Cut 0.10.1

### PR DESCRIPTION
**Pull request by bwplotka**
@GiedriusS reverted the fix metric name regression in examples as they are really ONLY examples, so no point in adding them to patch release IMO (:

### CODE REVIEW

**Comment by brancz** [comment]
I'd be ok with keeping the mixin part of releases, but don't feel too strongly.

**Comment by metalmatze** [comment]
Having the mixin as part of releases would definitely elevate their importance and thus hopefully increase their quality. Afterall those alerts are what people are going to rely on moving forward.

**Comment by bwplotka** [comment]
Why? No one look on those on release branch - everyone do that on the master, right :thinking:

**Comment by brancz** [comment]
We tend to use the same mixin version of etcd, ceph, etc. as the version of the software itself. I feel it would be useful to version along, then we can try to do changed to the mixin whenever we do changes on master, that way it's easy to know which mixin is compatible with which version.

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.10.0
```
Comment (squat) [inline VERSION]: :p

**Comment by bwplotka** [comment]
Fair points @brancz @metalmatze let's touch it on our Thanos sync. We need to decide then if e.g we make a mistake and discrepancy between mixin and code appear, will it require patch release?

**Pull request merged**